docs/configuration.md

```diff
@@ -7,7 +7,7 @@ Prettier uses [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) for co
 
 - A `.prettierrc` file, written in YAML or JSON, with optional extensions: `.yaml/.yml/.json/.js`.
```

Comment:
We should remove the `.js` here since we've mentioned it in the following item.

Reply:
Done!